test-utils/testlib/src/standard_test_cases.rs

```diff
@@ -840,6 +840,30 @@ pub fn test_delete_key_last(node: impl Node) {
     assert!(node_user.get_access_key(&account_id, &node.signer().public_key()).is_err());
 }
 
+fn assert_access_key(
+    access_key: &AccessKey,
+    access_key_view: AccessKeyView,
+    result: &FinalExecutionOutcomeView,
+    user: Box<dyn User>,
+) {
+    let key = {
+        #[cfg(feature = "protocol_feature_access_key_nonce_range")]
+        {
+            let mut key = access_key.clone();
+            let block = user.get_block_by_hash(result.transaction_outcome.block_hash);
+            if let Some(b) = block {
+                key.nonce = (b.header.height - 1) * AccessKey::ACCESS_KEY_NONCE_RANGE_MULTIPLIER;
+            }
+            key
+        }
+        #[cfg(not(feature = "protocol_feature_access_key_nonce_range"))]
+        {
+            let _ = (user, result);
```

Comment:
I am not sure what is happening here, are we trying to avoid user and result being not used?

Reply:
Yes